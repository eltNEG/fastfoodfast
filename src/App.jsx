import React from 'react';

export default () => <div>FastFoodFast</div>;
