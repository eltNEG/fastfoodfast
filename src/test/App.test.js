import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('App.jsx', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
