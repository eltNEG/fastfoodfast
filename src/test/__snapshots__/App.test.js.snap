// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.jsx should match snapshot 1`] = `
<BrowserRouter>
  <NavBar />
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      key="home"
      path="/"
    />
    <Route
      component={[Function]}
      key="not-found"
      path="*"
    />
  </Switch>
  <_default />
</BrowserRouter>
`;
